server/pkg/api/authenticationHandlers.go

```diff
@@ -26,6 +26,8 @@ func (ctx *Context) logoutRoute(w http.ResponseWriter, r *http.Request) {
 func loginHandler(ctx *Context, w http.ResponseWriter, r *http.Request) {
 	username, password, ok := r.BasicAuth()
 
+	rememberMe := r.URL.Query().Get("RememberMe")
```

Comment:
We'll have to refactor this into a POST call. This shouldn't be a query parameter. My bad.

I'll create an issue for this.